test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -449,16 +457,19 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			}
 
 			By("Exposing TCP 80 internally on the php-apache deployment")
-			err = phpApacheDeploy.Expose("ClusterIP", 80, 80)
-			Expect(err).NotTo(HaveOccurred())
-			_, err = service.Get(longRunningApacheDeploymentName, "default")
-			Expect(err).NotTo(HaveOccurred())
+			s, _ := service.Get(longRunningApacheDeploymentName, "default")
+			if s == nil {
+				err := phpApacheDeploy.Expose("ClusterIP", 80, 80)
+				Expect(err).NotTo(HaveOccurred())
+				_, err = service.Get(longRunningApacheDeploymentName, "default")
+				Expect(err).NotTo(HaveOccurred())
+			}
 		})
 
 		It("should have stable external container networking as we recycle a bunch of pods", func() {
 			name := fmt.Sprintf("alpine-%s", cfg.Name)
 			command := fmt.Sprintf("nc -vz 8.8.8.8 53 || nc -vz 8.8.4.4 53")
-			successes, err := pod.RunCommandMultipleTimes(pod.RunLinuxPod, "alpine", name, command, cfg.StabilityIterations)
+			successes, err := pod.RunCommandMultipleTimes(pod.RunLinuxPod, "alpine", name, command, cfg.StabilityIterations, 1*time.Second, retryCommandsTimeout)
```

Comment:
`pod.RunCommandMultipleTimes` now takes in a retry interval and timeout to accommodate "get" retries